lib/brackr/distance_calculator.rb

```diff
@@ -0,0 +1,16 @@
+module Brackr
+  module Distance
+    def calculate_distance(dest_latlng, current_lat, current_long)
```

Comment:
this method seems to be doing a lot, could we break it down into smaller methods?